init.lua

```diff
@@ -222,9 +249,9 @@ minetest.register_craft({
 minetest.register_craft({
 	output = 'trash_can:dumpster',
 	recipe = {
-		{'default:coalblock', 'default:coalblock', 'default:coalblock'},
-		{'default:steel_ingot', 'dye:dark_green', 'default:steel_ingot'},
-		{'default:steel_ingot', 'default:steel_ingot', 'default:steel_ingot'},
+		{ moditems.coal_item, moditems.coal_item, moditems.coal_item},
+		{ moditems.iron_item, moditems.green_dye, moditems.iron_item},
+		{ moditems.iron_item, moditems.iron_item, moditems.iron_item},
```

Comment:
Please use either of these variants:
`{moditems.iron_item, moditems.iron_item, moditems.iron_item},` or
`{ moditems.iron_item, moditems.iron_item, moditems.iron_item },`

Reply:
Done, ty.